resources/pages/SuccessUpload.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Let's use this page since we have it)
```js
console.log('File uploaded');

res.sendFile(path.join(__dirname, '../resources/pages/SuccessUpload.html'));
```